src/GafferImageUI/OpenColorIOAlgo.cpp

```diff
@@ -347,7 +347,12 @@ IECoreGL::Shader::SetupPtr displayTransformToFramebufferShader( const OCIO_NAMES
 			unsigned height = 0;
 			OCIO_NAMESPACE::GpuShaderDesc::TextureType channel = OCIO_NAMESPACE::GpuShaderDesc::TEXTURE_RGB_CHANNEL;
 			OCIO_NAMESPACE::Interpolation interpolation = OCIO_NAMESPACE::INTERP_LINEAR;
+#if OCIO_VERSION_HEX >= 0x02030000
+			OCIO_NAMESPACE::GpuShaderDesc::TextureDimensions dimensions;
```

Comment:
[TextureDimensions](https://opencolorio.readthedocs.io/en/latest/api/shaders.html#PyOpenColorIO.GpuShaderCreator.TextureDimensions) is an enum with values of TEXTURE_1D and TEXTURE_2D. Do we need to do anything differently depending on the value? Perhaps the `(height > 1) ? GL_TEXTURE_2D : GL_TEXTURE_1D;` heuristic at L378 should be replaced? Oh, and also the same in the misleadingly-named `AllocateTexture2D()`.

Relevant reading I discovered while trying to figure out what this TextureDimensions thing was all about :

https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1762
https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1758

Reply:
After a bit of discussion we think there's no apparent immediate value in addressing this so I've added a todo to replace the various `height` tests with `dimensions` once we can do it cleanly after dropping support for OCIO 2.2.